megamek/src/megamek/client/Client.java

```diff
@@ -701,7 +694,7 @@ protected void receiveAttack(Packet c) {
 
 
     // Should be private?
-    public String receiveReport(Vector<Report> v) {
+    public String receiveReport(List<Report> v) {
```

Comment:
What was the impetus behind changing the Vector<Report> to a List<Report>?

If we're doing this, we should also change the variable name to "l" or something more representative, and replace strings that reference "report vector" with "report list" for consistency.

Reply:
List is an interface and therefore much more versatile and more modern than Vector as a parameter. Also Vector is synchronized which is slower and we dont need that (wont make much of a difference, just saying). But will rename the parameter.